includes/Widgets/Stories.php

```diff
@@ -321,7 +321,6 @@ public function form( $instance ) {
 				'id'            => 'image_alignment',
 				'name'          => 'image_alignment',
 				'label'         => __( 'Image Alignment', 'web-stories' ),
-				'type'          => 'checkbox',
```

Comment:
Maybe alignment options should be a dropdown / select menu?

Reply:
This can be explored in a new ticket.  It's a radio button everywhere at the moment I think. Not changing this here, just removing some unused arg.